src/python/grpcio_tests/tests/stress/client.py

```diff
@@ -110,10 +110,19 @@ def _get_channel(target, args):
     channel_credentials = grpc.ssl_channel_credentials(
         root_certificates=root_certificates)
     options = (('grpc.ssl_target_name_override', args.server_host_override,),)
-    return grpc.secure_channel(
-        target, channel_credentials, options=options)
+    channel = grpc.secure_channel(target, channel_credentials, options=options)
   else:
-    return grpc.insecure_channel(target)
+    channel = grpc.insecure_channel(target)
+
+  # waits to the channel to be ready before we start sending messages
```

Comment:
Ahh I did not know of that call. I was mimicking the pattern I saw [here](https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests/qps/benchmark_client.py#L71).

I will use the simpler call.